src/Avalonia.Visuals/Animation/RenderLoopClock.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Avalonia.Rendering;
+
+namespace Avalonia.Animation
+{
+    public class RenderLoopClock : ClockBase, IRenderLoopTask
```

Comment:
Might a better name for this be `GlobalClock` as it's what's exposed from `Clock.Global`, right?

Reply:
I prefer the name `RenderLoopClock` since it's a clock that is pulsed by the render loop. We could also have a clock that's pulsed by time (i.e. the pre-this-PR timing), or something else.